### DISCUSSION TITLE
How to output attenuation A_v for high-z galaxies

### DISCUSSION THREAD

**Discussion by W-Chang-astro** [mboquien/cigale#38] [Q&A] [answered]
Hi, I am using CIGALE to fit SED. And I hope to get A_v from CIGALE. Should I directly output the parameter attenuation.V_filter? Or do I have to calculate the wavelength for the rest-frame V band? My galaxies are at 2.5<z<3 redshift. Thanks!

**Comment by mboquien** [comment, marked as answer]
If you are looking for the rest-frame V-band attenuation, you simple need to give the `V_B90` filter in the list of filters for the attenuation module. It should be included by default. If you want the Bayesian estimate, in the pdf_analysis module, you can add `attenuation.V_B90`. I think it should be sufficient.

**Reply by W-Chang-astro** [reply]
Hi, Thanks for your quick reply! I just want to confirm, so the parameter "attenuation.filter" always stands for the rest-frame band?

**Reply by mboquien** [reply]
Yes, that is rest-frame.

**Reply by W-Chang-astro** [reply]
Got it. Thanks so much!
